src/views/forms/FormTransactionBase/FormTransactionBase.ts

```diff
@@ -177,8 +182,13 @@ export class FormTransactionBase extends Vue {
     /**
      * it creates the deadlines for the transactions.
      */
-    protected createDeadline(): Deadline {
-        return Deadline.create(this.epochAdjustment);
+    protected createDeadline(deadlineInHours = 2): Deadline {
```

Comment:
Just wondering, if can just apply the solution here. 
we do not need involved in the Store state.

Example

```ts
protected async createDeadline(deadlineInHours = 2): Deadline {
    return  await (await DeadlineService.create(repositoryFactory)).createDeadlineUsingServerTime(deadlineInHours);
}
```

```ts
// to call
await this.createDeadline(x hours)
```